api/src/lib/s3.ts

```diff
@@ -17,4 +17,15 @@ if (process.env.STAGE === 'local') {
     };
 }
 
+type OctopusBucketType = 'images' | 'pdfs' | 'sitemaps';
+const buckets: {
+    [key in OctopusBucketType]?: string;
+} = {};
+
+for (const nameSegment of ['images', 'pdfs', 'sitemaps']) {
```

Comment:
Not a huge fan of hardcoded array values like these without any type checking. 
nameSegment could be anything...

You can convert type OctopusBucketType to enum and then easily loop through it or simply remove the optional key in buckets and define the bucket for each key there, there are just 3 buckets.

Reply:
Yeah, my initial idea to loop through since most of the name is repeated for each bucket ended up being more mess than it's worth. I've ended up defining it simply as you suggested